extensions/blocks/podcast-player/edit.js

```diff
@@ -256,14 +257,29 @@ const PodcastPlayerEdit = ( {
 	return (
 		<>
 			<BlockControls>
-				<ToolbarGroup>
-					<Button
-						aria-label={ __( 'Edit Podcast Feed URL', 'jetpack' ) }
-						onClick={ () => setIsEditing( true ) }
-					>
-						{ __( 'Replace', 'jetpack' ) }
-					</Button>
-				</ToolbarGroup>
+				{ /* @todo Fallback can be removed when WP 5.4 is the minimum supported version. */ }
+				{ ToolbarGroup ? (
+					<ToolbarGroup>
+						<Button
+							aria-label={ __( 'Edit Podcast Feed URL', 'jetpack' ) }
+							onClick={ () => setIsEditing( true ) }
+						>
+							{ __( 'Replace', 'jetpack' ) }
+						</Button>
+					</ToolbarGroup>
+				) : (
+					<Toolbar
+						controls={ [
+							{
+								title: __( 'Edit Podcast Feed URL', 'jetpack' ),
```

Comment:
Will this and the `Replace` strings be required for translation twice if we call it twice? Or are they _merged_ somehow and then translated as one?

Reply:
It shouldn't be a problem as same strings are consolidated from the whole Jetpack codebase, backend and frontend. We have scripts in place that extract all strings from codebase and unique ones will be uploaded to GlotPress for translation.